src/gamedata/unit.rs

```diff
@@ -45,28 +75,28 @@ pub struct Unit {
     m_Angle :f32,
     m_DontAttackPerson :u64,
     m_DontAttackForceMask :i32,
-    m_ItemList :u64,
+    pub m_ItemList : &'static UnitItemList,
     m_ItemSelected :u64,
     m_AccessoryList :u64,
     pub m_GodUnit :Option<&'static GodUnit>,
-    m_GodLink :Option<&'static GodUnit>,
+    pub m_GodLink :Option<&'static GodUnit>,
     pub m_Ring :Option<&'static UnitRing>,
     m_ExtraSight :i32,
-    m_MoveDistance :i32,
-    m_MaskSkill :u64,
-    m_EquipSkill :u64,
-    m_PrivateSkill :u64,
-    m_ReceiveSkill :u64,
-    m_SupportedSkill :u64,
-    m_EquipSkillPool :u64,
-    m_LearnedJobSkill :u64,
+    pub m_MoveDistance :i32,
+    pub m_MaskSkill : Option<&'static SkillArray>,
+    pub m_EquipSkill :&'static SkillArray,
+    pub m_PrivateSkill :&'static SkillArray,
+    pub m_ReceiveSkill :&'static SkillArray,
+    pub m_SupportedSkill :&'static SkillArray,
+    pub m_EquipSkillPool :&'static SkillArray,
+    pub m_LearnedJobSkill :&'static SkillArray,
     m_OriginalAptitude :u64,
-    m_Aptitude :u64,
-    m_WeaponMask :u64,
+    pub m_Aptitude : &'static mut WeaponMask,
```

Comment:
L o w e r c a s e, and no m_

Reply:
None of the unit field variables are renamed from the original unit.rs. Should probably stay unchanged for capability.